.github/actions/setup-stellar/action.yaml

```diff
@@ -29,12 +29,14 @@ runs:
       if: steps.cache-stellar.outputs.cache-hit != 'true'
       run: |
         mkdir -p stellar-binaries
-        curl -L -o stellar-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz https://github.com/stellar/stellar-cli/releases/download/v${{ inputs.stellar-version }}/stellar-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz
-        curl -L -o soroban-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz https://github.com/stellar/stellar-cli/releases/download/v${{ inputs.stellar-version }}/soroban-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz
+        curl -L -o stellar-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz https://github.com/stellar/stellar-cli/releases/download/v${{ inputs.stellar-version }}/stellar-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz        
         tar -xvf stellar-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz -C stellar-binaries
-        tar -xvf soroban-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz -C stellar-binaries
         rm -rf stellar-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz
-        rm -rf soroban-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz
+
+        curl -L -o soroban-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz https://github.com/stellar/stellar-cli/releases/download/v${{ inputs.stellar-version }}/soroban-cli-${{ inputs.stellar-version }}-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
soroban-cli-*.tar.gz file is required to download, unzip and remove it? could we avoid downloading it?

Reply:
we could remove it entirely, but my idea is to allow the gh action to work with previous releases where the soroban client is still around